src/Symfony/Component/Validator/Constraint.php

```diff
@@ -27,6 +27,7 @@
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
```

Comment:
Why adding this here ? The abstract class cannot be used as an attribute. And child classes defining concrete constraints will have to specify the proper target anyway.

Reply:
(removed in #61649 BTW)